content/scripts/build.js

```diff
@@ -639,23 +633,24 @@ class Builder {
     fs.mkdirSync(folderpath, { recursive: true });
   }
 
-  ensurePopularities() {
-    if (!Object.keys(this.allPopularities).length) {
-      const { popularitiesfile } = this.options;
-      if (popularitiesfile) {
-        this.allPopularities = JSON.parse(
-          fs.readFileSync(popularitiesfile, "utf8")
-        );
-
-        this.logger.info(
-          chalk.magenta(
-            `Parsed ${Object.keys(
-              this.allPopularities
-            ).length.toLocaleString()} popularities.`
-          )
-        );
-      }
+  _getAllPopularities() {
```

Comment:
I kept it as a method even though there's only 1 single place it gets called from. That's because the length of the `ensureAllTitles` method is already too long.